webui/react/src/pages/TrialDetails/TrialDetailsMetrics.tsx

```diff
@@ -119,6 +113,9 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
 
   const chartsProps = useMemo(() => {
     const out: ChartsProps = [];
+    if (!metricsLoaded) {
+      return NotLoaded;
+    }
```

Comment:
**nit:* can have this before even `out` gets defined above

Reply:
✅